tools/certs/generate-istioctl-certs.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Generates files for istioctl's --cert-dir directory.
+
+set -o errexit
+
+if [[ "x$DEBUG_SHELL_SCRIPT" != "x" ]]; then
+   set -x
+fi
+
+if [ -z "${ISTIO_NAMESPACE}" ]; then
+   export ISTIO_NAMESPACE=istio-system
+   echo '$ISTIO_NAMESPACE not set, defaulting to' $ISTIO_NAMESPACE
+fi
+
+if [ -z "${CERT_DIR}" ]; then
+   export CERT_DIR=istioctl-certs
+   echo '$CERT_DIR not set, defaulting to' $CERT_DIR
+fi
+
+SECRET_NAME=istio-ca-secret
+if ! kubectl -n "${ISTIO_NAMESPACE}" get secret "${SECRET_NAME}" > /dev/null ; then
+   SECRET_NAME=cacerts
+fi
+
+echo ""
+echo Getting Istiod root certificate from secret "${SECRET_NAME}" in "${ISTIO_NAMESPACE}"
```

Comment:
would be good to put in instruction on which cluster to point to when running this cmd.  I think it is the data plane cluster which has istio-ca-secret